### ISSUE TITLE
Generated API doc for query_region has a wrong description

### ISSUE THREAD

**Issue by bmatthieu3** [astropy/astroquery#1470]
See https://astroquery.readthedocs.io/en/latest/api/astroquery.vizier.VizierClass.html#astroquery.vizier.VizierClass.query_region

The description of `query_region` (which is generated with its API doc) is identical to the description of `query_region_async`.

**Comment by bsipocz** [comment]
The difference between `query_region` and `query_region_async` is indeed whether they return a Table or the response object. And the docs is inherited from `query_region_async`. Could you please clarify what's wrong with it, you mean the first sentence (which probably shouldn't be inherited)?

**Comment by keflavich** [comment]
That's it exactly.  The docstring for `query_region` says it does something different from... `query_region`.  Oops!

**Comment by bmatthieu3** [comment]
@bsipocz Yes the title of the doc is good but the first description sentence is the same as `query_region_async`

I also found the same issue for simbad: https://astroquery.readthedocs.io/en/latest/api/astroquery.simbad.SimbadClass.html#astroquery.simbad.SimbadClass.query_region.

**Comment by bsipocz** [comment]
oh. Then I suppose there will be many other affected modules due to the templating.
